web/app/components/input/__field/input__field.scss

```diff
@@ -3,7 +3,7 @@
   $fontSize: 16px;
   $paddingVrt: 10px;
   $lines: 3;
-  $height: $fontSize * $lineHeight * $lines + $paddingVrt * 2;
+  $height: calc($fontSize * $lineHeight * $lines+$paddingVrt * 2);
```

Comment:
Isn't it just works with simple-vars plugin?

Reply:
No. We get this:
```
height:16px*1.4*3 + 10px*2;
```